include/kllvm/ast/AST.h

```diff
@@ -84,9 +84,11 @@ class KOREMetaSortVariable : public KOREMetaSort {
 };
 
 enum class SortCategory {
-  Map, List, Set, Int, Float, StringBuffer, Bool, MInt, Symbol
+  Uncomputed, Map, List, Set, Int, Float, StringBuffer, Bool, MInt, Symbol
```

Comment:
I assume Uncomputed is equivalent to some kind of "sort variable"?

Reply:
It just means we haven't computed the category yet for this term.